### PR TITLE
feat: add section id for methodology

### PR DESCRIPTION
**Pull request by PeterMosmans**
Apply opinionated XML formatting

### CODE REVIEW

**Review by skyanth — CHANGES_REQUESTED** [review]
Pentestmethodologie, not Pentest methodologie
https://www.spatiegebruik.nl/regels.html#:~:text=Samenstellingen%20worden%20in%20het%20Nederlands,samengestelde%20woord%20is%20altijd%20fout.

File: xml/source/snippets/offerte/nl/methodology.xml [hunk 1]
```diff
@@ -1,54 +1,45 @@
...
-
-<p><b>Stap 6: Exploitatie</b><br/>
+  <p><b>Stap 6: Exploitatie</b><br />
 Hier wordt gepoogd om kwetsbaarheden in de verscheidene componenten uit te buiten.
 De diverse applicaties en componenten van de infrastructuur van de klant worden
 grondig gecontroleerd op vaak voorkomende ontwerp-, configuratie- en programmeerfouten.</p>
-
-<p>Opmerking: <company_long/> gebruikt open-source scanning tools als basis, maar 
+  <p>Opmerking: <company_long
+    /> gebruikt open-source scanning tools als basis, maar
```
Comment (gronke) [inline xml/source/snippets/offerte/nl/methodology.xml:37]: There should be no line break here.

File: xml/source/snippets/offerte/nl/methodology.xml [hunk 2]
```diff
@@ -1,54 +1,45 @@
...
 voert in de regel de meeste exploitatiepogingen handmatig uit.</p>
-
-<p><b>Stap 7: Rapportage</b><br/>
+  <p><b>Stap 7: Rapportage</b><br />
 Na het afronden van de test zal een rapport worden geleverd waarin de resultaten en ontdekte kwetsbaarheden stap voor stap worden beschreven. Het rapport en de resultaten worden eventueel gepresenteerd aan de verantwoordelijke projectleider of -manager in het kantoor van de klant.</p>
-
-<p>Stappen 4-6 kunnen meerdere malen herhaald worden per test. Zo kan er bijvoorbeeld door exploitatie van kwetsbaarheden toegang worden
+  <p
+  >Stappen 4-6 kunnen meerdere malen herhaald worden per test. Zo kan er bijvoorbeeld door exploitatie van kwetsbaarheden toegang worden
```
Comment (gronke) [inline xml/source/snippets/offerte/nl/methodology.xml:42]: Please remove the line break here as well.

**Review by gronke — CHANGES_REQUESTED** [review]

**Comment by PeterMosmans** [comment]
@skyanth @gronke :
I've fixed the spelling error (and another one :)), and applied opinionated Prettier formatting.

**Pull request merged**